test/elements/mixer_test.exs

```diff
@@ -36,32 +36,32 @@ defmodule MixerTest do
       end
     end
 
-    options = %Pipeline.Options{
-      elements: [
-        source1: %Source{output: {@first_input_frames, generator}, caps: %Frame{encoding: :utf8}},
-        source2: %Source{output: {@second_input_frames, generator}, caps: %Frame{encoding: :utf8}},
-        mixer: Mixer,
-        sink: Sink
-      ],
-      links: [
-        link(:source1) |> via_in(:first_input) |> to(:mixer),
-        link(:source2) |> via_in(:second_input) |> to(:mixer),
-        link(:mixer) |> to(:sink)
-      ]
-    }
+    spec = [
+      child(:source1, %Source{
+        output: {@first_input_frames, generator},
+        stream_format: %Frame{encoding: :utf8}
+      })
+      |> via_in(:first_input)
+      |> child(:mixer, Mixer)
+      |> child(:sink, Sink),
+      child(:source2, %Source{
```

Comment:
[NIT] I believe you could use anynomous children there: https://hexdocs.pm/membrane_core/Membrane.ChildrenSpec.html#child/2 
(except for `:sink`)

Reply:
I'll leave the spec as is and add a note about anonymous children in the tutorial